layouts/partials/footer.html

```diff
@@ -40,6 +41,7 @@
 
         </div>
```

Comment:
Hi adrianmo,

I've done it.

But I'm not sure that this is the change you're waiting for.

I'm in doubt 'cause just the above code block, which refers to 'About us' section, enclose div tag between if's...

Greats.